RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This is an internal change to give every ``Example`` object a label. It should
```

Comment:
Best not to mention internal interfaces, and in this case I think it could help with shrinking in some circumstances .  Eg:

> This patch may improve example shrinking slightly for some strategies.